unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -239,10 +239,10 @@ export class CreatorLockForm extends React.Component {
       unlimitedKeys,
       valid,
     } = this.state
-
+    const lockAddress = lock ? lock.address : ''
     // NOTE: maxNumberOfKeys must be a text input in order to support the infinity symbol
     return (
-      <FormLockRow>
+      <FormLockRow className="lockForm" data-address={lockAddress}>
```

Comment:
Multiple locks could be in "edit" mode. This allows us to make sure we use the right form in the integration tests at least.